README.md

```diff
@@ -1,14 +1,28 @@
 ![Open Service Broker API](https://www.openservicebrokerapi.org/wp-content/uploads/2016/12/osbapi_logo_concept3_wtm.png)
 
 ## Latest Release: v2.12
+
 [Read the specification](https://github.com/openservicebrokerapi/servicebroker/blob/v2.12/spec.md)
 
 [Release Notes for past versions](release-notes.md)
 
-## Working Changes
+## SDLC
+
+### Working Changes
+
 [Read the working changes to the spec](spec.md)
```

Comment:
I know you didn't add this, but can we remove it - the above sentence makes no sense to me.